### PR TITLE
Adds mkdocs service to docker compose

### PR DESCRIPTION
**Pull request by jefftriplett**
Quick overview of how to run docs:

```shell
docker-compose --profiles=docs up
```

This will run mkdocs on port 4000

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -94,6 +99,25 @@ services:
...
+    command: sh -c "mkdocs serve -f docs/mkdocs.yml --dev-addr 0.0.0.0:4000"
+    entrypoint: ""
+    environment:
+      FAST_MODE: "true"
+      LIVE_RELOAD_SUPPORT: "true"
+    init: true
+    profiles:
+      - docs
+    depends_on:
+      - django
```
Comment (saadmk11) [inline docker-compose.yml:115]: Can we omit this?

**Review by saadmk11 — COMMENTED** [review]
Just tried it out looks great! :100:

File: docker-compose.yml (hunk 1 above)
Comment (jefftriplett) [inline docker-compose.yml:115]:
```suggestion
```

Comment (jefftriplett) [reply docker-compose.yml]: done

**Pull request merged**